Test/dafny0/AssumptionVariables0.dfy.expect

```diff
@@ -1,5 +1,5 @@
 AssumptionVariables0.dfy(7,31): Error: there may be at most one assignment to an assumption variable, the RHS of which must match the expression "a0 && <boolean expression>"
-AssumptionVariables0.dfy(8,35): Error: there may be at most one assignment to an assumption variable, the RHS of which must match the expression "a2 && <boolean expression>"
+AssumptionVariables0.dfy(8,42): Error: there may be at most one assignment to an assumption variable, the RHS of which must match the expression "a2 && <boolean expression>"
```

Comment:
The error was here.
```
    ghost var a1, {:assumption} a2 := true, false;  // error
                                   ^^
```
Now it is here
```
    ghost var a1, {:assumption} a2 := true, false;  // error
                                          ^
```
I'm not very familiar with assumptions, but I just read it was meant to cache verification at the Boogie level.
https://www.microsoft.com/en-us/research/wp-content/uploads/2016/12/krml245.pdf
That said, I think I the error should have been on neither of those places above but on the variable "a2" itself, as it must be assigned "true" and not "false".
If you are unsure about what is best, could you please at least revert to the previous behavior? I don't think highlighting a comma is meaningful anyway.